### PR TITLE
bpo-26389: Allow passing an exception object in the traceback module

### PR DESCRIPTION
**Pull request by ZackerySpytz**
The format_exception(), format_exception_only(), and
print_exception() functions can now take an exception object
as a positional-only argument.

Co-Authored-By: Matthias Bussonnier <bussonniermatthias@gmail.com>

https://bugs.python.org/issue26389

### CODE REVIEW

File: Doc/library/traceback.rst [hunk 1]
```diff
@@ -62,6 +67,10 @@ The module defines the following functions:
    .. versionchanged:: 3.5
       The *etype* argument is ignored and inferred from the type of *value*.
 
+   .. versionchanged:: 3.10
+      The *etype* parameter has been renamed to *exc* and is now
```
Comment (pablogsal) [inline Doc/library/traceback.rst:71]: Could you add a simple sentence regarding this to `Porting to Python 3.10` in the 3.10 what's new?

**Review by pablogsal — APPROVED** [review]
LGTM

I left a minor comment

**Comment by ZackerySpytz** [comment]
@pablogsal Hi, Pablo. Can this PR be merged?

**Comment by pablogsal** [comment]
> @pablogsal Hi, Pablo. Can this PR be merged?

Thanks for the ping @ZackerySpytz

**Pull request merged**